src/app/app-components/login/login-page.component.html

```diff
@@ -1,13 +1,25 @@
 <div class="login-container">
-    <div class="container">
-        <span class="login-text">Login</span>
-        <span>E-mail:</span>
-        <input [(ngModel)]="email" type="email" placeholder="example@mail.com">
-        <span>Password:</span>
-        <input [(ngModel)]="password" type="password" placeholder="*********">
-        <div class="actions">
-            <a href="/">Forgot password?</a>
-            <button (click)="onLogin()" class="login-button">Login</button>
+    <form [formGroup]="formGroup">
+        <div class="container">
+            <span class="login-text">Login</span>
+            <div class="row input-label" [class.error]="this.computeControlValidity(emailControl)">
```

Comment:
We don't need to use `this. ...` in the template.

Reply:
Oops.. my bad - I'll remove it before submitting.